.github/workflows/try-release.yml

```
@@ -16,6 +16,7 @@ jobs:
       matrix:
         GO_VERSION:
           - "1.21.5"
+          - "1.22"
```

You'll need to [replace the GPG public key](https://github.com/letsencrypt/boulder/blob/b483ec228047a93b9f25e432f442c1827227dc13/tools/fetch-and-verify-go.sh#L25-L224) with [this](https://dl.google.com/linux/linux_signing_key.pub) before the go1.22 try-release job succeeds.

Additionally, Go pulled a fast one on us as of the go1.21.0 release. Prior to that we'd have written `- "1.19"` or `- "1.18"`, but now it should be `- "1.22.0"`. See this URL: https://go.dev/dl/go1.22.0.linux-amd64.tar.gz

```suggestion
          - "1.22.0"
```